libc/config/darwin/arm/entrypoints.txt

```diff
@@ -94,6 +94,10 @@ set(TARGET_LIBC_ENTRYPOINTS
     libc.src.stdlib.calloc
     libc.src.stdlib.realloc
     libc.src.stdlib.free
+
+    # stdio.h external entrypoints
+    libc.src.stdio.vsprintf
```

Comment:
```suggestion
    libc.src.stdio.asprintf
```
I'm assuming you meant `asprintf`? `vsprintf` is already allocated and is the `va_list` variant of `sprintf`.

Reply:
what do you mean by sprint is already allocated? I don't think its in the entry point.txt

Reply:
I meant 'implemented' sorry. It's not in the `darwin` one so I guess you could add it? But that would probably be a separate path. All the string related `stdio.h` functions should be added to darwin, but we don't have a file IO interface yet, so `sprintf` works but not `printf`.